### ISSUE TITLE
Could you fix your website's CSS?

### ISSUE THREAD

**Issue by losh11** [Someguy123/LiteVault-iOS#2]
So some things are intersecting, some things don't look too good on mobile.
The first (initial) commit will just be a cheap port, running off your website's server (like an iframe).

But what I really want to do is to make everything natively, I've got a JSOpen SDK which allows me to execute your JS code whilst everything being shown through Swift.

Here is what your broken website (on an iPhone) looks like:-
http://imgur.com/a/yDYTO

So if you could fix everything, that would be great!

**Comment by Someguy123** [comment]
@losh11 What exactly is "broken" in that screenshot? It's not obvious to me; is it the same in mobile Safari?

**Comment by losh11** [comment]
According to iOS tradition, on the first page (the first image) there should not be any text showing 'secure...' unless the user scrolls down. So only the login and register part should be shown up till the point the user scrolls down.

In the second picture, it's kinda obvious that there is something wrong with the nav bar, everything is exploded into three rows. It should all be on one row and have CSS, making it look like a nav bar. Again, on the same image, the BTC and LTC address are intersecting. 

So basically stuff like that. This is pretty much just safari showing everything.

**Comment by Someguy123** [comment]
@losh11 I'm not sure how I would even align those buttons like you suggest without making it look worse on other devices i.e. with media tags. The current design was done by someone else, as I am pretty terrible at anything design related like this. If you can explain how I could do that, or even just submit a PR with adjusted CSS that would be great.

On the other note, having that tab bar in one row would mean the user has to horizontally scroll the navigation, which is kind-of awkward. I'm not sure of the best way of dealing with that, I haven't actually had anyone complain about it, and personally I find it convenient on my Android device. If you have an idea that fixes that without inhibiting the UX then I would be happy to implement that.

The LTC and BTC addresses I'll fix right now, not sure how that slipped through, but easy fix there.

**Update:** LTC and BTC addresses should be fixed. If not, wait a few minutes due to Cloudflare caching
